aiogram/fsm/storage/base.py

```diff
@@ -19,6 +19,68 @@ class StorageKey:
     destiny: str = DEFAULT_DESTINY
 
 
+class KeyBuilder(ABC):
+    """Base class for key builder."""
+
+    @abstractmethod
+    def build(self, key: StorageKey, part: Literal["data", "state", "lock"]) -> str:
+        """
+        This method should be implemented in subclasses
+
+        :param key: contextual key
+        :param part: part of the record
+        :return: key to be used in storage's db queries
+        """
+        pass
+
+
+class DefaultKeyBuilder(KeyBuilder):
+    """
+    Simple key builder with default prefix.
```

Comment:
Docstring should started without linebreak

```suggestion
    """Simple key builder with default prefix.
```

Reply:
Ah, other code uses the same format. Skipped.